.github/workflows/ci.yml

```diff
@@ -0,0 +1,166 @@
+name: Continuous Integration
+on:
+  pull_request:
+  merge_group:
+    types: [checks_requested]
+  workflow_dispatch:
```

Comment:
`workflow_call` should be needed for it to work as a child of `cd.yml`:
```suggestion
  workflow_dispatch:
  workflow_call:
```

Reply:
Good catch, added